integration-cli/docker_cli_export_import_test.go

```diff
@@ -18,12 +18,6 @@ func (s *DockerSuite) TestExportContainerAndImportImage(c *check.C) {
 		c.Fatal("failed to create a container", out, err)
 	}
 
-	inspectCmd := exec.Command(dockerBinary, "inspect", containerID)
```

Comment:
the problem with waitRun is that I think its racey for cases like this. If the container goes from  creating->running->stopped (because the cmd we run is so quick - which `true` is) then waitInspect could miss it.